src/test/regress/expected/inherit.out

```diff
@@ -2023,16 +1971,17 @@ select * from inhpar;
 -- Also check ON CONFLICT
 insert into inhpar as i values (3), (7) on conflict (f1)
   do update set (f1, f2) = (select i.f1, i.f2 || '+');
+ERROR:  modification of distribution columns in OnConflictUpdate is not supported
```

Comment:
Maybe add a comment that the error is an expected behavior in GPDB?

Reply:
added